### PR TITLE
Remove the ability to have a null lab

### PR DESCRIPTION
**Pull request by meadsteve**
Not sure how you feel about this but it didn't make any sense to me to allow an experiment without a lab. I couldn't see any usages of this.

### CODE REVIEW

**Comment by meadsteve** [comment]
@daylerees brackets removed! coding style matched.

**Comment by daylerees** [comment]
Checkstyle error on CI mate, but I'd be happy to merge on pass. Nice one!

**Comment by meadsteve** [comment]
@daylerees well that's an embarrassing mistake to have made. Fixed!

**Comment by daylerees** [comment]
Haha no worries. I've been guilty of the same from time to time. Cheers bud. :)

File: src/Experiment.php [hunk 1]
```diff
@@ -68,9 +68,9 @@ class Experiment
      * Create a new experiment.
      *
      * @param string                     $name
-     * @param \Scientist\Laboratory|null $laboratory
+     * @param \Scientist\Laboratory $laboratory
```
Comment (daylerees) [inline src/Experiment.php:71]: Could you align the parameter names here please mate?

File: src/Laboratory.php [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Laboratory
     /**
      * Collection of journals to report to.
      *
-     * @var array
+     * @var Journal[]
```
Comment (daylerees) [inline src/Laboratory.php]: FQN here please mate.

File: tests/InternTest.php [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <?php
 
 use Scientist\Intern;
+use Scientist\Laboratory;
```
Comment (daylerees) [inline tests/InternTest.php]: Downward sloping namespaces please... I know... I'm terrible. Sorry!

**Comment by meadsteve** [comment]
Cool. I'll take a look at the issues tomorrow. I'm not sure I'll be able to bring myself to do downward sloping namespaces though :-)

**Comment by meadsteve** [comment]
and done.

**Comment by daylerees** [comment]
Thanks!

**Pull request merged**